htdocs/product/dynamic_price/class/price_parser.class.php

```diff
@@ -156,11 +156,10 @@ public function parseExpression($product, $expression, $values)
 
 		//Retrieve all extrafield for product and add it to values
 		$extrafields = new ExtraFields($this->db);
+		$extralabels = $extrafields->fetch_name_optionals_label($product->table_element);
```

Comment:
I made a little change (pushed after this PR) because it is better to use the already loaded $extrafields object if available.